### ISSUE TITLE
Cannot play certain games

### ISSUE THREAD

**Issue by GoogleCodeExporter** [WhiteNeo/wiiflow#180]
```
What steps will reproduce the problem?
1. A 4.1U Wii with cios 249 rev 19 and cios 222 rev 5.1, cios 222 rev 4 38 
merge 37, cios 224 rev 5 38 base 57
2. Trying to boot up Star Wars the Force Unleashed, Metroid Prime Trilogy, 
Metroid Prime 3 Corruption, and Super Smash Bros Brawl

What is the expected output? What do you see instead?
The screen should show loading game, but instead crashes.

What version of the product are you using? On what operating system?
r43 249, on FW 4.1U

Please provide any additional information below.
It seems theres a problem playing dual layer discs, because the only two 
that I have installed, MPTrilogy and SSBBrawl dont start. 

```

Original issue reported on code.google.com by `techfrea...@hotmail.com` on 5 Jun 2010 at 3:51

**Comment by GoogleCodeExporter** [comment]
```
You can't run games have more this size 4.0 gb ¿you hdd is in format ntfs 
right? only a 
heard in format fat32 don't have problems with this game i'am use configurable 
usb 
loader
```

Original comment by `Digital....@gmail.com` on 5 Jun 2010 at 6:07

**Comment by GoogleCodeExporter** [comment]
```
222 rev 5.1, cios 222 rev 4 38 
merge 37, cios 224 rev 5 38 base 57?

i wanna know how you have 2 different versions of 222 installed at the same 
time......  
Look up some guides about how to get mpt and ssbb playing.
```

Original comment by `Miigotu` on 5 Jun 2010 at 6:10

**Comment by GoogleCodeExporter** [comment]
```
Sorry for the confusion, its cios 223 that is rev 4 38 merge 37...

And i use Configurable USB as well, just thought I would test Wiiflow and see 
the 
compatibility. So far no go, because those four games dont work while they play 
fine in 
CFG on pretty much any cIOS. Even USB GX loads those games fine, on default 
settings. 
```

Original comment by `techfrea...@hotmail.com` on 6 Jun 2010 at 11:27

**Comment by GoogleCodeExporter** [comment]
```
games with this error for me were Okami, Dead Space , Metroid trilogy, new u 
fitness, little kings story and Sakura Wars.  Do not play intro music at game 
launch 
screen, code dumps if I try to change alt dol entry in settings and code dumps 
if 
try to launch game. The six largest games in my HD.

4.2E, cios 249 rev19 (tried a few bases), hermes v4 222 or 223, same error with 
beta 
versions 222 and 249.  On a single NTFS partition HD
```

Original comment by `notsoflu...@hotmail.com` on 7 Jun 2010 at 8:52

**Comment by GoogleCodeExporter** [comment]
```
I've had this issue too. Its caused by wbfs files larger than 4gb. You an 
either split 
them into 4Gb chunks, or wait for a fix.
```

Original comment by `hanimka...@gmail.com` on 7 Jun 2010 at 9:29

**Comment by GoogleCodeExporter** [comment]
```
Oh okay, i forgot i had games in bigger chunks. 

When I first switched to ntfs, i set the split size to infinite for copying my 
games 
off the WBFS partition, but since then ive had it at 4GB. 

I dont believe its worth the effort, probably just stick with CFG for now.

Thanks for the answer!
```

Original comment by `techfrea...@hotmail.com` on 7 Jun 2010 at 5:18

**Comment by GoogleCodeExporter** [comment]
```
Okay, so i did change them all after all, and it fixes them all but Super Smash 
Bros Brawl which now says:

Disc_SetWBFS failed

The game still plays fine with CFG, im gonna try re-installing from the disc 
later.
```

Original comment by `techfrea...@hotmail.com` on 7 Jun 2010 at 7:46

**Comment by GoogleCodeExporter** [comment]
```
I fixed this for 2.0.. We now use a new libntfs.
```

Original comment by `Miigotu` on 19 Jun 2010 at 10:40
- Changed state: **Fixed**
